.github/CODEOWNERS

```diff
@@ -703,15 +718,15 @@
 # TODO(#13808): Revert ownership to @vojtechjelinek after 2021-09-14.
 /core/templates/karma.module.ts @DubeySandeep
 # TODO(#13596): Add @krishnarao22 back to codeowners after 2021-09-01
-/core/tests/protractor_utils/ @kevintab95 @SAEb-ai @U8NWXD
-/core/tests/protractor_desktop/ @kevintab95 @SAEb-ai @U8NWXD
-/core/tests/protractor/ @kevintab95 @SAEb-ai @U8NWXD
-/core/tests/protractor.conf.js @kevintab95 @SAEb-ai @U8NWXD
-/extensions/interactions/*/protractor.js @kevintab95 @U8NWXD
-/extensions/interactions/protractor.js @kevintab95 @U8NWXD
-/extensions/rich_text_components/*/protractor.js @kevintab95 @U8NWXD
-/extensions/rich_text_components/protractor.js @kevintab95 @U8NWXD
-/extensions/objects/protractor.js @kevintab95 @U8NWXD
+/core/tests/protractor_utils/ @SAEb-ai @U8NWXD
```

Comment:
Ditto

Reply:
Yup it's a permanent change per discussion with the Automated QA lead.

Reply:
Thanks for the info!